lib/symbols.js

```diff
@@ -5,6 +5,7 @@ const keys = {
   kChildren: Symbol('fastify.children'),
   kServerBindings: Symbol('fastify.serverBindings'),
   kBodyLimit: Symbol('fastify.bodyLimit'),
+  kAcceptedHTTPMethods: Symbol('fastify.acceptedHTTPMethods'),
```

Comment:
kSupportedHTTPMethods?

Reply:
I think we don't support an http method (eg: `MKCALENDAR`), but we accept/approve/ingest/process/"do not block" the HTTP method and the user write the business logic to support it.

Reply:
Now the getter is wrong, I just copy-paste the existing one 😄